compose-builder/tui-generator.sh

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env bash
+
+# /*******************************************************************************
+#  * Copyright 2022 
```

Comment:
Copyright needs an entity. Your company?

Reply:
I am committing as a stand-alone contributor here. Should I then add my own name or are there different policies for contribution?

Reply:
Have added my e-mail address, as a personal contribution